examples/using/using.ino

```diff
@@ -0,0 +1,31 @@
+/*
+  This example demonstrate pH value reading
+*/
+#include "Arduino.h"
+
+// Include library
+#include "TroykaPH.h"
+
+TroykaPH phMeter(A4); // set used analog pin
+
+uint32_t showingTime;
+constexpr uint32_t INTERVAL = 3000;
+
+void setup() {
+    phMeter.begin(); // if module not calibrated or...
+    // phMeter.begin(correction,zeroShift); // if you have it (use calibrate.ino) for it
+
+    Serial.begin(9600);
+    showingTime = millis() + INTERVAL; // show result once per 3 seconds
+}
+
+void loop() {
+    phMeter.update(1000); // real read from sensor once per second
+    // (you can increase this period, in practice pH value changing too slowly)
+
+    if (millis() - showingTime > INTERVAL) {
+        Serial.print("\nCurrent pH: ");
+        Serial.print(phMeter.read(), 1);
+        showingTime += INTERVAL;
+    }
+}
```

Comment:
How the example works? Why not do it in our standard way?

```cpp
void loop() {
    // Read the sensor data values PH
    phMeter.read();
    // Receive the stored PH
    float valuePH = phMeter.getPH();
    // Print results
    Serial.print("PH Value = ");
    Serial.println(valuePH);
    // Wait 1000 ms
    delay(1000);
}
```

Reply:
It makes no sense to constantly poll the pH sensor. First, its meaning changes very slowly. Secondly, it's a long time, there are a lot of delays when reading. Therefore, it makes sense to read from time to time, and return the last read value upon request. As here.